typescript: Cannot find module 'tslib'

**Description**

When importing `@mcap/nodejs` (and other packages), I'm getting an error `Cannot find module 'tslib'`.  


- Version: v22.10.0
- Platform: macOS

**Steps To Reproduce**


Create a minimal `package.json`:
```json
{
  "dependencies": {
    "@mcap/nodejs": "^1.0.1"
  }
}
```

Install dependencies and import module
```bash
npm install
node -e "require('@mcap/nodejs')"
```

**Expected Behavior**

😎 

**Actual Behavior**

```bash
node -e "require('@mcap/nodejs')"
node:internal/modules/cjs/loader:1252
  throw err;
  ^

Error: Cannot find module 'tslib'
Require stack:
- /Users/ben/tmp/mcap/node_modules/@mcap/nodejs/dist/cjs/src/index.js
- /Users/ben/tmp/mcap/[eval]
    at Function._resolveFilename (node:internal/modules/cjs/loader:1249:15)
    at Function._load (node:internal/modules/cjs/loader:1075:27)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:218:24)
    at Module.require (node:internal/modules/cjs/loader:1340:12)
    at require (node:internal/modules/helpers:141:16)
    at Object.<anonymous> (/Users/ben/tmp/mcap/node_modules/@mcap/nodejs/dist/cjs/src/index.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Object..js (node:internal/modules/cjs/loader:1689:10)
    at Module.load (node:internal/modules/cjs/loader:1318:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/ben/tmp/mcap/node_modules/@mcap/nodejs/dist/cjs/src/index.js',
    '/Users/ben/tmp/mcap/[eval]'
  ]
}

Node.js v22.10.0
```


----

## Additional information

The contents of `node_modules/@mcap/nodejs/dist/cjs/src/index.js` imports `tslib` for the `__exportStar` helper.

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const tslib_1 = require("tslib");
tslib_1.__exportStar(require("./FileHandleReadable"), exports);
tslib_1.__exportStar(require("./FileHandleWritable"), exports);
//# sourceMappingURL=index.js.map
```

This looks like it's been added by the [`importHelpers` option](https://www.typescriptlang.org/tsconfig/#importHelpers), which is present in the [base foxglove/tsconfig](https://github.com/foxglove/tsconfig/blob/main/base.json#L12).

It'd be great if tslib was a dependency, or the helpers were inlined?

## Workaround

Installing `tslib` separately will resolve this issue.

```bash
npm install tslib
```

--- a/typescript/browser/package.json
+++ b/typescript/browser/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mcap/browser",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "Support library for using MCAP in the browser",
   "license": "MIT",
   "repository": {
@@ -53,5 +53,8 @@
     "prettier": "3.1.0",
     "ts-jest": "29.1.1",
     "typescript": "5.2.2"
+  },
+  "dependencies": {
+    "tslib": "^2.5.0"
   }
 }

--- a/typescript/nodejs/package.json
+++ b/typescript/nodejs/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mcap/nodejs",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Support library for using MCAP in Node.js",
   "license": "MIT",
   "repository": {
@@ -54,5 +54,8 @@
     "prettier": "3.1.0",
     "ts-jest": "29.1.1",
     "typescript": "5.2.2"
+  },
+  "dependencies": {
+    "tslib": "^2.5.0"
   }
 }

--- a/typescript/support/package.json
+++ b/typescript/support/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@mcap/support",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Common decompression for use with MCAP files",
   "license": "MIT",
   "repository": {
@@ -57,6 +57,7 @@
     "@foxglove/wasm-bz2": "^0.1.1",
     "@foxglove/wasm-lz4": "^1.0.2",
     "@foxglove/wasm-zstd": "^1.0.1",
-    "protobufjs": "^7.2.5"
+    "protobufjs": "^7.2.5",
+    "tslib": "^2.5.0"
   }
 }

